starboard/android/shared/audio_renderer_passthrough.cc

```diff
@@ -308,13 +311,23 @@ SbTime AudioRendererPassthrough::GetCurrentMediaTime(bool* is_playing,
   *is_underflow = false;  // TODO: Support underflow
   *playback_rate = playback_rate_;
 
+  // Align the reported seek time with the timestamp of the first written audio.
```

Comment:
This covers a case where there may be a time regression after the first audio is written.

For example, if we seek to timestamp 303 ms, and GetCurrentMediaTime() is called before the first audio packet is written, we don't know the timestamp of the audio. If we assume each packet covers 32 ms, we can predict the first timestamp to be 288 ms. After the first audio packet is written, we can use the correct timestamp. The alternative would be to use the unadjusted seek time, which could cause a time regression after we use the timestamp from the first written audio packet.

However, predicting the timestamp relies on the fact that we expect each written sync frame to be 32 ms, which may not always be the case. This is an edge case so I don't think it will affect much, but I would like to get all of your thoughts on this.